me.timschneeberger.jdsp4linux.yaml

```diff
@@ -0,0 +1,48 @@
+app-id: me.timschneeberger.jdsp4linux
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: jamesdsp
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-run/pipewire-0:ro
+  - --device=dri
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.freedesktop.portal.Background
+cleanup:
+  - "*.a"
+  - "*.la"
+  - "*.h"
+  - "*.m4"
+  - "*.pm"
+  - /include
+  - /lib/pkgconfig
+  - /share/pkgconfig
+modules:
+  - name: jamesdsp
+    buildsystem: qmake
+    modules:
+      - jamesdsp-modules.yaml
```

Comment:
the *content* of this should be inserted above line 25.

It's more maintainable.